doc/library/config.txt

```diff
@@ -636,6 +636,18 @@ import theano and print the config variable, as in:
 
     If ``'no_check'``, assume present and the version between header and library match (so less compilation at context init)",
 
+.. attribute:: config.dnn.include_path
+
+    Default: ``''``
```

Comment:
I would put the phrase in parenthesis here. Something like:

Default: cuda_root/include or ''